### DISCUSSION TITLE
user account has been blocked

### DISCUSSION THREAD

**Discussion by empato-limited** [vercel/community#3334] [Help] [answered]
### Summary

our account has been blocked and we cannot login to open a ticket ... we currently have around 50 sites offline? where can we ask for support?

### Steps to Reproduce

user empato-limited

**Comment by amyegan** [comment, marked as answer]
Please email [registration@vercel.com](mailto:registration@vercel.com) and include the email and username for the account. To protect against platform abuse, Vercel employs multiple layers of security and verification. Providing additional information about your account and the project you are deploying can help accelerate this process.
